SasquatchMac/SasquatchMac/ServiceStateStore.swift

```diff
@@ -0,0 +1,26 @@
+import Foundation;
+
+@objc class ServiceStateStore : NSObject {
+  private static let AnalyticsKey : String = "kMSAnalyticsIsEnabledKey";
+  private static let CrashesKey : String = "kMSCrashesIsEnabledKey";
+
+  class var AnalyticsState : Bool {
+    get {
+      return UserDefaults.standard.bool(forKey: AnalyticsKey);
+    }
+    set {
```

Comment:
We shouldn't set any values to NSUserDefaults for the key that is used by the SDK. Get should be enough.